remote-access/access-over-Internet/README.md

```diff
@@ -2,10 +2,39 @@
 
 You can connect to your Raspberry Pi over the internet from another computer or a mobile device. There are a number of ways to do this, which we cover below.
 
-## Port forwarding
+## Port forwarding (IPv4)
 
 One method is to set up port forwarding on your router. To do this, you must change the configuration of your router to forward all inbound traffic from the internet on a specific port to the local IP address of your Raspberry Pi. Most routers have this feature available. However, every router is different so you will need to consult your router's user manual for instructions. The settings can be tricky if your Pi is behind a firewall or if there is more than one router. One disadvantage of port forwarding is that it exposes a network port on your private LAN to the public internet. This is a known security vulnerability and must be managed carefully.
 
+## Introduction for IPv6 newcomers
+
+There are big differences in handling IPv4 and IPv6 for remote access in residential environments(dsl,cable,fibre, maybe mobile too).
+With IPv4 it was and is still very common to use the public address of the router in combination with port forwarding. The router is the only device with a public address. So the router gets a packet and translates it (address and port) to a private IP address on the lan. Also the router is (mostly) responsible for getting a (nice) DNS name.
+With IPv6 some things have changed. Depending on your isp (different types of ds-lite and dualstack) you don't get a public IPv4 address anymore.
+Instead of one public IPv4 address you get a whole network with IPv6 addresses (prefix).
```

Comment:
What does "(prefix)" mean here?

Reply:
My raspberry pi 2 has
`2001:a61:475:6601:ba27:ebff:feb6:f293`
My raspberry pi 4 has
`2001:a61:475:6601:dea6:32ff:fe23:6be1`
They share the same prefix `2001:a61:475:6601::/64` I got that prefix from my ISP ( in reality it is /56, but some providers give more, some only /64)

Reply:
Ahh, in that case maybe it's worth expanding "(prefix)." to something like "(with a common prefix)." ?